tephra-hbase-compat-0.98/src/test/java/co/cask/tephra/hbase98/TransactionAwareHTableTest.java

```diff
@@ -363,6 +363,34 @@ public void testAbortedTransactionalDelete() throws Exception {
   }
 
   @Test
+  public void testColumnFamilyDelete() throws Exception {
+    HTable hTable = createTable(Bytes.toBytes("TestColFamilyDelete"), new byte[][]{TestBytes.family});
+    try (TransactionAwareHTable txTable = new TransactionAwareHTable(hTable, TxConstants.ConflictDetection.ROW)) {
+      TransactionContext txContext = new TransactionContext(new InMemoryTxSystemClient(txManager), txTable);
+      txContext.start();
+      txTable.put(new Put(TestBytes.row).add(TestBytes.family, TestBytes.qualifier, TestBytes.value));
+      txContext.finish();
+
+      // Start a tx, delete the row and then abort the tx
+      txContext.start();
+      txTable.delete(new Delete(TestBytes.row));
```

Comment:
One more thing you could do to enhance this test case is to have multiple column families, but write data only to one. And then issue a row delete with a rollback. Later do a scan with Visibiltity.SNAPSHOT_ALL from PR#94 to make sure there are no delete markers present in any column families.